pkg/accounts/model/repository.ts

```diff
@@ -91,6 +91,7 @@ export interface AccountAvatarRepository {
     mediumID: MediumID,
   ): Promise<Result.Result<Error, void>>;
   findByID(accountID: AccountID): Promise<Result.Result<Error, Medium>>;
+  findByIDs(accountIDs: AccountID[]): Promise<Result.Result<Error, Medium[]>>;
```

Comment:
```suggestion
  findByIDs(accountIDs: readonly AccountID[]): Promise<Result.Result<Error, Medium[]>>;
```
読んでもらう配列の引数は `readonly` を指定したほうがよいと思います. `AccountHeaderRepository` の方も同様です.